hi-ml-histopathology/pytest.ini

```diff
@@ -6,3 +6,4 @@ log_cli_level = DEBUG
 addopts = --strict-markers
 markers =
     fast: Tests that should run very fast, and can act as smoke tests to see if something goes terribly wrong.
+    gpu: Tests that require at least 1 GPU to be present
```

Comment:
If the test is marked with `pytest.mark.gpu` but not with `@pytest.mark.skipif(no_gpu,...)`,  should it still run on a machine with no GPUs?

Reply:
good point, fixed.